src/Smart.Design.Razor/TagHelpers/TableOfContent/TableOfContentHtmlGenerator.cs

```diff
@@ -23,6 +24,11 @@ public TagBuilder GenerateListOfItems(List<string> titles)
                 li.Attributes["aria-selected"] = "true";
             }
 
+            if (i == 0 || i == 1)
+            {
+                li.AddCssClass("not-numbered");
+            }
+
```

Comment:
Du chipotage mais à la place de double condition, moi je l'aurais mis comme ça pour la lisibilité. Après tu utilises une ligne de plus dans ma solution donc à voir XD 
```suggestion
   if (i == 0)
            {
                li.Attributes["aria-selected"] = "true";
                li.AddCssClass("not-numbered");
            }

            else if (i == 1)
            {
                li.AddCssClass("not-numbered");
            }
```